components/filter/filter-tags.js

```diff
@@ -8,7 +8,7 @@ import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
  * A tag-list allowing the user to see (and remove) the currently applied filters.
  */
 
-const CLEAR_TIMEOUT = 210; /** Corresponds to timeout in _dispatchChangeEvent in filter + 10 ms */
+const CLEAR_TIMEOUT = 310; /** Corresponds to timeout in _dispatchChangeEvent in filter + 10 ms */
```

Comment:
Any reason for the jump from 200 to 300? Did the extra amount cut down on HM calls even more than adding just the debouncing?

Reply:
I changed to 300s as normally when selecting too fast, it seemed to be good enough time to select multiple filters, that will reduce the triggered events. This would just cut the calls to update the list in non-HM. In case of HM, lot of calls are associated with toggling the filters and then updating the list which can be reduced!